### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by linhaoqi027** [ultralytics/yolov5#839]
## ❔Question

how can i change lr0?

**Comment by glenn-jocher** [comment]
@linhaoqi027 lr0 is a hyperparameter. You modify it in the hyperparameter yaml files, depending on whether you are finetuning i.e.  `python train.py --weights yolov5s.pt` or training from scratch i.e. `python train.py --weights ''`

https://github.com/ultralytics/yolov5/blob/4fb8cb353f7d1945e3e1b270980c883c82297d2f/data/hyp.finetune.yaml#L6

https://github.com/ultralytics/yolov5/blob/4fb8cb353f7d1945e3e1b270980c883c82297d2f/data/hyp.scratch.yaml#L6

**Comment by linhaoqi027** [comment]
thx,i got that

**Comment by linhaoqi027** [comment]
I am writing Tutorials about converting yolov5 to openvino,how to write 
![image](https://user-images.githubusercontent.com/46484513/91788037-caee5a80-ec3d-11ea-9eea-877a5c3279ca.png) like you. So It can help me write Tutorials

**Comment by glenn-jocher** [comment]
@linhaoqi027 you just right click on the line number you want to quote in github to produce that link.

**Comment by bdeng3** [comment]
There are three different learning rates (lr0, lr1, lr2) in the result.csv file. Are they representing learning rate for different layers (such as lr0: conv layer, lr2: batchnorm layer)? And it looks like we only need to worry about tuning lr0 in the hyper-parameter file?

**Comment by glenn-jocher** [comment]
@bdeng3 no, `lr0` hyp is initial learning rate, `lr` in results.csv is parameter group lr. Parameter groups are defined in train.py:
https://github.com/ultralytics/yolov5/blob/fcb225c1c81a17fbaeff027b1f4be5300049e8a8/train.py#L137-L155
